### ISSUE TITLE
.ps1 doesn't have an unblock option and the powershell code won't do anything

### ISSUE THREAD

**Issue by kosmosterrl** [Whitebrim/VSCode-PlaydateTemplate#12]
I don't know if this is a windows 11 problem or something but the .ps1 properties doesn't have an unblock option and whenever I put in the powershell code it does nothing.
![Screenshot 2023-08-23 142342](https://github.com/Whitebrim/VSCode-PlaydateTemplate/assets/143037809/b93fc114-d286-4946-b89a-8ecc9cf929ec)
![Screenshot 2023-08-23 142456](https://github.com/Whitebrim/VSCode-PlaydateTemplate/assets/143037809/43375163-1c4e-4600-9e66-114628546c35)

**Comment by z80andrew** [comment]
I had the same problem on Win10. Rather than set an execution policy, I unblocked the specific file.

In Visual Studio Code:
Terminal -> New Terminal (Opens a PowerShell prompt at the current project location)

Run `Unblock-File '.\Build and Run (Simulator).ps1'`

**Comment by Panegyr** [comment]
> I had the same problem on Win10. Rather than set an execution policy, I unblocked the specific file.
> 
> In Visual Studio Code: Terminal -> New Terminal (Opens a PowerShell prompt at the current project location)
> 
> Run `Unblock-File '.\Build and Run (Simulator).ps1'`

This should be the recommended method in the readme for allowing vscode to execute the script, changing the entire execution policy for one script is quite excessive and could lead to being exploited by malware

**Comment by Whitebrim** [comment]
> > I had the same problem on Win10. Rather than set an execution policy, I unblocked the specific file.
> > 
> > In Visual Studio Code: Terminal -> New Terminal (Opens a PowerShell prompt at the current project location)
> > 
> > Run `Unblock-File '.\Build and Run (Simulator).ps1'`
> 
> This should be the recommended method in the readme for allowing vscode to execute the script, changing the entire execution policy for one script is quite excessive and could lead to being exploited by malware

As far as I know unblocking .ps1 file is not the same as running it with admin rights. Both should be done to run it successfully. In the readme there is a code that lets this script be executed without need to launch VS Code as admin (recommend via pull request).
There's a nasty bug in VS Code if it's launched with admin rights: https://github.com/microsoft/vscode/issues/91613

**Comment by Panegyr** [comment]
> > > I had the same problem on Win10. Rather than set an execution policy, I unblocked the specific file.
> > > In Visual Studio Code: Terminal -> New Terminal (Opens a PowerShell prompt at the current project location)
> > > Run `Unblock-File '.\Build and Run (Simulator).ps1'`
> > 
> > 
> > This should be the recommended method in the readme for allowing vscode to execute the script, changing the entire execution policy for one script is quite excessive and could lead to being exploited by malware
> 
> As far as I know unblocking .ps1 file is not the same as running it with admin rights. Both should be done to run it successfully. In the readme there is a code that lets this script be executed without need to launch VS Code as admin (recommend via pull request). There's a nasty bug in VS Code if it's launched with admin rights: [microsoft/vscode#91613](https://github.com/microsoft/vscode/issues/91613)

Yes this is correct actually, I spoke too soon, RemoteSigned should be safe enough as it's the default setting for Windows Server Installations

**Comment by Panegyr** [comment]
As for OP, the greyed out Read Only / Hidden / System properties is very odd. 

Upon further inspection it looks like they're examining the file as it exists in the ZIP archive, and it hasn't been extracted, indicated by the Method and CRC32 values, it looks like they may have simply confused the default icon for a zip file in windows with that of an actual folder and tried to run the shell script inside of it, which wouldn't work at all afaik
